.babelrc.js

```diff
@@ -16,5 +16,10 @@ module.exports = {
     ['@babel/proposal-object-rest-spread', { loose }],
     ['transform-react-remove-prop-types', { mode: 'unsafe-wrap' }],
     'macros'
-  ]
+  ],
+  env: {
```

Comment:
this section kinda duplicates what is already specified in main babel configuration, would be IMHO better to reuse what's already there - I have also seen babel's `env` property rather unintuitive (do u know how those env-specific plugins/presets work in relation to common presets/plugins?)

I would prefer adding `@babel/transform-commonjs-modules` as devDep and adding it to common plugins in such way
```js
const test = process.env.NODE_ENV === 'test'

module.exports = {
  // ... config
  plugins: [
    // ... common plugins
    test && '@babel/transform-commonjs-modules'
  ].filter(Boolean)
}
```

Reply:
I was trying to remove the config on `@babel/env` here so it would default to commonjs for the modules property.